ggml/include/ggml.h

```diff
@@ -1445,6 +1447,22 @@ extern "C" {
             float                 beta_fast,
             float                 beta_slow);
 
+    GGML_API struct ggml_tensor * ggml_mrope_ext(
```

Comment:
What is the logic for the "m" in "mrope"? Is it coming from "multi-rope" to indicate it is multidimensional?

If so, a better name would be `ggml_rope_multi` or `ggml_rope_nd`

Reply:
According to the paper, it was originally "Multimodal Rotary Position Embedding." However, I believe "multidimensional RoPE" would be a clearer name. Therefore, I'll go with `ggml_rope_nd`.

Reply:
The `test-rope.cpp` additions are great. Need to also add tests to `test-backend-ops` and we can proceed to merge.

Reply:
Test cases for `test-backend-ops` have also been added.